providers/catalog.go

```diff
@@ -156,6 +157,25 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 		},
 	},
 
+	// Copper configuration
+	Copper: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.copper.com/developer_api/v1",
```

Comment:
Can you please not add the version in the Base URL? If they release a new version, we'll have to update it.